core/src/main/java/org/geysermc/geyser/translator/protocol/java/JavaClientboundResourcePacksPacket.java

```diff
@@ -38,6 +38,6 @@ public class JavaClientboundResourcePacksPacket extends PacketTranslator<Clientb
     @Override
     public void translate(GeyserSession session, ClientboundResourcePackPacket packet) {
         // We need to "answer" this to avoid timeout issues related to resource packs
-        session.sendDownstreamPacket(new ServerboundResourcePackPacket(ResourcePackStatus.DECLINED));
+        session.sendDownstreamPacket(new ServerboundResourcePackPacket(ResourcePackStatus.SUCCESSFULLY_LOADED));
```

Comment:
Are we really able to just skip the `ACCEPTED` step?

Reply:
Looks like it works! https://discord.com/channels/613163671870242838/613168464634576897/1159621179594776636

One idea though; we could check for the `required` boolean in the `ClientboundResourcePackPacket` & only claim that we accepted the pack if it is being forced.